src/components/NavBar/index.js

```diff
@@ -100,23 +98,13 @@ const Left = styled.div`
   ${mediumAndUp`width: 50%;`};
 `;
 
-const MessageContainer = styled.div`
-  background-color: rgba(0, 0, 0, 0.2);
-  text-align: center;
-  font-size: ${typography.size.hecto};
-  font-weight: ${typography.weight.regular};
-  padding-top: 2px;
-  padding-bottom: 2px;
-`;
-
 const NavBar = ({
   children,
   position,
   invert,
   className,
   backgroundColor,
   style,
-  message,
```

Comment:
I'm weary to make this change since it will not be backwards compatible for other developers that are utilizing this component... Perhaps it would be better to just not use the `message` prop within our integration?

Reply:
Ok, then, probably, we should leave aurora as it is